src/test/java/uk/gov/register/functional/ApplicationTest.java

```diff
@@ -30,7 +30,7 @@ public void setup() {
     @Test
     public void appSupportsCORS() {
         String origin = "http://originfortest.com";
-        Response response = register.target().path("/entries")
+        Response response = register.targetRegister("address").path("/entries")
```

Comment:
`getRequest()` is a convenience wrapper around `targetRegister()` for GET requests.

I'm in two minds. I think `getRequest` reads nicer but it doesn't do very much. /shrug